tasks/main.yml

```diff
@@ -15,32 +15,61 @@
     repo: '{{ docker__upstream_repository }}'
     state: 'present'
     update_cache: True
+  register: docker__register_apt_repo
```

Comment:
@tallandtree This variable doesn't seem to be used anywhere else, can it be removed?